<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(188, 160, 115);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="usernameLable">
       <property name="styleSheet">
        <string notr="true">color:rgb(80, 53, 39);
font: 87 18pt &quot;Source Code Pro Black&quot;;</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameInput">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro Black</family>
         <pointsize>14</pointsize>
         <weight>10</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(222, 222, 222);
font: 87 14pt &quot;Source Code Pro Black&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="passwordLable">
       <property name="styleSheet">
        <string notr="true">color:rgb(80, 53, 39);
font: 87 18pt &quot;Source Code Pro Black&quot;;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordInput">
       <property name="font">
        <font>
         <family>Source Code Pro Black</family>
         <pointsize>14</pointsize>
         <weight>10</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(222, 222, 222);
font: 87 14pt &quot;Source Code Pro Black&quot;;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
     <item row="0" column="0">
      <widget class="QPushButton" name="loginButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(222, 222, 222);
font: 87 14pt &quot;Source Code Pro Black&quot;;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="newAccountButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(222, 222, 222);
font: 87 14pt &quot;Source Code Pro Black&quot;;</string>
       </property>
       <property name="text">
        <string>Create new</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="loginMessageText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 87 14pt &quot;Source Code Pro Black&quot;;</string>
    </property>
    <property name="text">
     <string>Error messages go here.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
